third_party/kythe/proto/CAMPFIRE

```diff
@@ -28,6 +28,40 @@
     }
   },
   {
+    "name": "claim_proto",
```

Comment:
Although it is true the protos are not used in Shipshape, I would rather bring in entire packages from Kythe instead of editing CAMPFIRE files to remove portions of a package.